staging/prometheus-operator/patch/mesosphere/templates/hooks/prometheus-cluster-id-hooks.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     app: {{ template "kube-prometheus-stack.name" . }}-prometheus
 {{ include "kube-prometheus-stack.labels" . | indent 4 }}
   annotations:
-    helm.sh/hook: pre-install
+    helm.sh/hook: pre-install,pre-delete
```

Comment:
I have a few questions about this, including checking kba testing as why this was not an issue prior. Still without the hook the SA would be available right?

Reply:
I don't think this happens often, but in some scenarios perhaps the SA was removed prematurely causing the pre-delete jobs to fail. The SA is also created here https://github.com/mesosphere/charts/blob/master/staging/prometheus-operator/templates/prometheus/serviceaccount.yaml - perhaps it would be cleaner for our mesosphere hooks to create and use a different SA?